s3cortxutils/s3confstore/setup.py

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+import os
+from setuptools import setup
+files = ["VERSION"]
+
+# Load the version
+s3confstore_version = "1.0.0"
```

Comment:
do we need to set this by default as 1.0.0?

Reply:
We used same as is done in s3cipher.

What would be the suggestion from your side ?

Reply:
we are fetching it from VERSION file so do we need to set it explicitly ?